.github/workflows/pytest_gpu.yml

```diff
@@ -0,0 +1,217 @@
+          ec2-image-id: ami-0759580dedc953d1f
+          ec2-instance-type: g4dn.xlarge
+          subnet-id: subnet-0983be43
+          security-group-id: sg-4cba0d08
+          aws-resource-tags: >
+            [
+              {"Key": "Name", "Value": "ludwig-github-${{ github.head_ref || github.sha }}"},
+              {"Key": "GitHubRepository", "Value": "${{ github.repository }}"},
+              {"Key": "GitHubHeadRef", "Value": "${{ github.head_ref }}"},
+              {"Key": "GitHubSHA", "Value": "${{ github.sha }}"}
+            ]
+
+  pytest-gpu:
+    if: needs.start-runner.result != 'skipped'
+    needs: start-runner # required to start the main job when the runner is ready
+    runs-on: ${{ needs.start-runner.outputs.label }} # run the job on the newly created runners
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        python-version: [3.8]
+        include:
+          - python-version: 3.8
+            pytorch-version: 1.13.1
+            torchscript-version: 1.10.2
+            ray-version: 2.0.0
+    env:
+      PYTORCH: ${{ matrix.pytorch-version }}
+      NEUROPOD_BASE_DIR: "/usr/local/lib/neuropod"
+      NEUROPOD_VERSION: "0.3.0-rc6"
+      TORCHSCRIPT_VERSION: ${{ matrix.torchscript-version }}
+      RAY_VERSION: ${{ matrix.ray-version }}
+      AWS_ACCESS_KEY_ID: ${{ secrets.LUDWIG_TESTS_AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.LUDWIG_TESTS_AWS_SECRET_ACCESS_KEY }}
+
+    name: py${{ matrix.python-version  }}, torch-${{ matrix.pytorch-version }}, gpu
+
+    timeout-minutes: 90
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Setup Linux
+        if: runner.os == 'linux'
+        run: |
+          sudo apt-get update && sudo apt-get install -y libsndfile1 cmake ccache build-essential g++-8 gcc-8
+          cmake --version
+
+      - name: Install CUDA drivers
+        run: |
+          wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/cuda-ubuntu2004.pin
+          sudo mv cuda-ubuntu2004.pin /etc/apt/preferences.d/cuda-repository-pin-600
+          wget https://developer.download.nvidia.com/compute/cuda/11.5.1/local_installers/cuda-repo-ubuntu2004-11-5-local_11.5.1-495.29.05-1_amd64.deb
+          sudo dpkg -i cuda-repo-ubuntu2004-11-5-local_11.5.1-495.29.05-1_amd64.deb
+          sudo apt-key add /var/cuda-repo-ubuntu2004-11-5-local/7fa2af80.pub
+          sudo apt-get update
+          sudo apt-get -y install cuda
+          # Downgrade NCCL to 2.11.4 to for compatibility with CUDA 11.6
+          # https://github.com/horovod/horovod/issues/3625#issuecomment-1309786643
+          sudo apt-get install --allow-change-held-packages --allow-downgrades -y libnccl2=2.11.4-1+cuda11.6 libnccl-dev=2.11.4-1+cuda11.6
```

Comment:
My hypothesis on why we get the [installation error](https://github.com/ludwig-ai/ludwig/actions/runs/4187196028/jobs/7256734164#step:5:54747) when you add this line is that these packages probably already exist in the base image from ray, but don't exists here. We're probably safe removing this. My bad!